pom.xml

```diff
@@ -13,8 +14,7 @@
 	<description>Aggregate Test and Job Results in a single HTML/email report.</description>
 	<url>https://github.com/jenkinsci/test-results-aggregator-plugin</url>
 	<properties>
-		<jenkins.version>2.277</jenkins.version>
-		<java.level>8</java.level>
+		<jenkins.version>2.332.1</jenkins.version>

```

Comment:
Needed for Mailer; the version of Mailer that has switched to Jakarta imports requires 2.332.1 or higher.